jdaviz/configs/default/plugins/export/tests/test_export.py

```diff
@@ -204,9 +206,10 @@ def test_export_data(cubeviz_helper, spectrum1d_cube):
     assert ep.data_invalid_msg == ''
 
 
-class TestExportPluginPlots():
+@pytest.mark.usefixtures('_jail')
```

Comment:
i don't think this needs this? the plots aren't actually being saved in these tests, since in the UI when you click to export they are downloaded rather than saved to the cwd

Reply:
Actually this is needed. Otherwise, you see 5 extra files left in the source dir after you do a local test run. And subsequent test run would fail because those files exist. And if you do not manually clean up those files, they might get accidentally checked into version control when a dev blindly do a `git add .` (as I tend to sometimes).

Reply:
Looks like the export plugin sets filename without path, so the plugin saves files in working dir (where pytest is invoked). I saw this for regions etc.

Reply:
ok, i don't understand why it works like this but i will trust you haha.

Reply:
Instead of using this fixture, we could also provide full path in the affected tests, but that would defeat the purpose of testing default output file values (which I am not sure was the original intention or not).

Reply:
i just don't understand what's being written out in those tests. nothing should be saved because its not in the browser

Reply:
I thought only viewer PNG goes through the browser. Region files use `regions`, etc.

Reply:
Ah, if you only want me to take it off the PNG, I can. I just slap it on all the test functions in that module for consistency.

Reply:
See bdbf8d09620e89397fda15b681d1f47afefa7ee5